http/router_utils.go

```diff
@@ -132,6 +143,34 @@ func ReadErrorKey(writer http.ResponseWriter, request *http.Request) (types.Erro
 	return types.ErrorKey(errorKey), true
 }
 
+// ReadRuleSelector retrieves the rule selector (rule_id|error_key) from request's
+// url or writes an error to writer.
+// The function returns the selector and a bool indicating if it was successful.
+func ReadRuleSelector(writer http.ResponseWriter, request *http.Request) (types.RuleSelector, bool) {
+	ruleSelector, err := GetRouterParam(request, "rule_id")
+	if err != nil {
+		const message = "unable to get rule FQDN"
+		log.Error().Err(err).Msg(message)
+		types.HandleServerError(writer, err)
+		return "0", false
```

Comment:
why `"0"`?

Reply:
really good question, no valid answer 🥲

Reply:
But I tested it with "", and change it. Marking conversation as resolved.